docs/content/contributing/maintainers-guidelines.md

```diff
@@ -7,128 +7,121 @@ description: "Interested in contributing more to the community and becoming a Tr
-but must have achieved several.
+The contributor should also meet one or several of the following requirements:
 
-- Enabled [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication) on your GitHub account
 - The contributor has opened and successfully run medium to large PR’s in the past 6 months.
+
 - The contributor has participated in multiple code reviews of other PR’s,
   including those of other maintainers and contributors.
-- The contributor showed a consistent pattern of helpful, non-threatening, and friendly behavior towards other community members in the past.
+
 - The contributor is active on Traefik Community forums
   or other technical forums/boards such as K8S slack, Reddit, StackOverflow, hacker news.
-- Have read and accepted the contributor guidelines.
+
+Any existing active maintainer can create an issue to discuss promoting a contributor to maintainer. 
+Other maintainers can vote on the issue, and if the quorum is reached, the contributor is promoted to maintainer.
+If the quorum is not reached within one month after the issue is created, it is closed.
 
 ## Maintainer's Responsibilities and Privileges
 
-There are lots of areas where you can contribute to the project,
-but we can suggest you start with activities such as:
-
-- PR reviewing.
-    - According to our guidelines we require you have at least 3 reviewers,
-      thus you can review a PR and leave the relevant comment if it is necessary.
-- Participating in a daily [issue triage](https://github.com/traefik/contributors-guide/blob/master/issue_triage.md).
-    - The process helps to understand and prioritize the reported issue according to its importance and severity.
-      This is crucial to learn how our users implement Traefik.
-      Each of the issues that are labeled as bug/possible bug/confirmed requires a reproducible use case. 
-      You can help in creating a reproducible use case if it has not been added to the issue
-      or use the sample code provided by the reporter.
-      Typically, a simple Docker Compose should be enough to reproduce the issue.
-- Code contribution.
-- Documentation contribution.
-    - Technical documentation is one of the most important components of the product.
-      The ability to set up a testing environment in a few minutes,
-      using the official documentation,
-      is a game changer.
-- You will be listed on our Maintainers GitHub page
-  and on our website in the section [maintainers](maintainers.md).
-- We will be promoting you on social channels (mostly on Twitter).
-
-## Governance
-
-- Roadmap meetings on a regular basis where all maintainers are welcome.
+As a maintainer, you are granted a vote for the following:
+
+- [PR review](https://github.com/traefik/contributors-guide/blob/master/pr_guidelines.md).
+
+- [Design review](https://github.com/traefik/contributors-guide/blob/master/proposal_guideines.md).
```

Comment:
the link for this is incorrect, it does not exist in current master or not in PR either